server/seed/seed.js

```diff
@@ -705,7 +705,7 @@ const generateRecord = function () {
 //Initiate randomly generated data - to increase/decrease amount of randomly generated records, alter the number itemIdTracker is being compared to
 /////////////////////////////////////
 if (process.env.node_env !== 'test') {
-  while (itemIdTracker <= 199) {
+  while (itemIdTracker <= 50099) {
```

Comment:
Why 50099?

Reply:
That is inherited seeder/generator that is now obsolete.  As for the answer?  got me.